packages/core/src/common/selection-command-handler.ts

```diff
@@ -28,17 +28,27 @@ export class SelectionCommandHandler<S> implements CommandHandler {
 
     execute(...args: any[]): Object | undefined {
         const selection = this.getSelection(...args);
-        return selection ? (this.options.execute as any)(selection, ...args) : undefined;
+        return selection ? (this.options.execute as any)(selection, ...args) : (this.options.execute as any)(undefined);
     }
 
     isVisible(...args: any[]): boolean {
         const selection = this.getSelection(...args);
-        return !!selection && (!this.options.isVisible || (this.options.isVisible as any)(selection as any, ...args));
+        //custom isVisible function overrides default checking mechanism
```

Comment:
@qx-juilliard comments must start with a `space`, it is the reason the CI is currently failing.
https://travis-ci.org/theia-ide/theia/jobs/564015232#L868